templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -535,7 +535,7 @@ Parameters:
   KeyPairName:
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
     Description: The EC2 Key Pair to allow SSH access to the instances.
-    Type: List<AWS::EC2::Keypair>
+    Type: List<AWS::ec2::KeyPair::KeyName>
```

Comment:
`AWS::ec2::KeyPair::KeyName` returns an object of all available KeyPairs so no need to wrap in a List. Should be defined like so:

`Type: AWS::ec2::KeyPair::KeyName`

Reply:
Done!